### DISCUSSION TITLE
Extras (Deleted scenes, making of, etc)

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-vue#146] [Features]

**Comment by AlfHou** [comment]
Any more description on this? Not quite sure what section this refers to.

**Comment by ferferga** [comment]
@AlfHou Are items associated to another item, useful for things like Makings of.

![image](https://user-images.githubusercontent.com/10274099/97735666-e7423280-1ada-11eb-86de-30062ff74d77.png)

See **Movie Extras** [here](https://support.emby.media/support/solutions/articles/44001159102-movie-naming)
